templates/migration-settings.py.erb

```diff
@@ -1,12 +1,22 @@
 PULP2_MONGODB = {
-    "name": "<%= scope['pulpcore::plugin::migration::mongo_db_name'] %>",
-    "seeds": "<%= scope['pulpcore::plugin::migration::mongo_db_seeds'] %>",
-    "username": "<%= scope['pulpcore::plugin::migration::mongo_db_username'] %>",
-    "password": "<%= scope['pulpcore::plugin::migration::mongo_db_password'] %>",
-    "replica_set": "<%= scope['pulpcore::plugin::migration::mongo_db_replica_set'] %>",
-    "ssl": "<%= scope['pulpcore::plugin::migration::mongo_db_ssl'] %>",
-    "ssl_keyfile": "<%= scope['pulpcore::plugin::migration::mongo_db_ssl_keyfile'] %>",
-    "ssl_certfile": "<%= scope['pulpcore::plugin::migration::mongo_db_ssl_certfile'] %>",
-    "verify_ssl": "<%= scope['pulpcore::plugin::migration::mongo_db_verify_ssl'] %>",
-    "ca_path": "<%= scope['pulpcore::plugin::migration::mongo_db_ca_path'] %>",
+    "name": "<%= @mongo_db_name %>",
+    "seeds": "<%= @mongo_db_seeds %>",
+<% if not @mongo_db_username.nil? -%>
+    "username": "<%= @mongo_db_username %>",
+<% end -%>
+<% if not @mongo_db_password.nil? -%>
+    "password": "<%= @mongo_db_password %>",
+<% end -%>
+<% if not @mongo_db_replica_set.nil? -%>
+    "replica_set": "<%= @mongo_db_replica_set %>",
+<% end -%>
+    "ssl": "<%= @mongo_db_ssl ? 'True' : 'False' %>",
```

Comment:
I think this should actually be a Python boolean
```suggestion
    "ssl": <%= @mongo_db_ssl ? 'True' : 'False' %>,
```

Reply:
and done.